### PR TITLE
[FLINK-5385] [core] Add a help function to create Row

### PR DESCRIPTION
**Pull request by wuchong**
Thanks for contributing to Apache Flink. Before you open your pull request, please take the following check list into consideration.
If your changes take all of the items into account, feel free to open your pull request. For more information and/or questions please refer to the [How To Contribute guide](http://flink.apache.org/how-to-contribute.html).
In addition to going through the list, please provide a meaningful description of your changes.

- [x] General
  - The pull request references the related JIRA issue ("[FLINK-XXX] Jira title text")
  - The pull request addresses only one issue
  - Each commit in the PR has a meaningful commit message (including the JIRA id)

- [ ] Documentation
  - Documentation has been added for new functionality
  - Old documentation affected by the pull request has been updated
  - JavaDoc for public methods has been added

- [x] Tests & Build
  - Functionality added by the pull request is covered by tests
  - `mvn clean verify` has been executed successfully locally or a Travis build has passed

Currently, it is trivial to create a Row, for example:

```java
Row row = new Row(3);
row.setField(0, "hello");
row.setField(1, true);
row.setField(2, 1L);
```

This PR introduces an `of` help method to create a Row, such as:

```java
Row row = Row.of("hello", true, 1L);
```

### CODE REVIEW

File: flink-core/src/main/java/org/apache/flink/types/Row.java [hunk 1]
```diff
@@ -113,4 +113,30 @@ public boolean equals(Object o) {
...
+	 * instead of
+	 * <pre>
+	 *     Row row = new Row(3);
+	 *     row.setField(0, "hello");
+	 *     row.setField(1, true);
+	 *     row.setField(2, 1L);
+	 * </pre>
+	 *
+	 */
+	public static Row of(Object... objects) {
```
Comment (fhueske) [inline flink-core/src/main/java/org/apache/flink/types/Row.java]: I would rename `objects` to `fields` or `values`

**Comment by fhueske** [comment]
good idea!
+1 to merge

File: flink-core/src/main/java/org/apache/flink/types/Row.java (hunk 1 above)
Comment (wuchong) [inline flink-core/src/main/java/org/apache/flink/types/Row.java]: good point!

**Comment by fhueske** [comment]
merging

**Pull request closed without merging**